crypto/keys/secp256k1/secp256k1.go

```diff
@@ -206,3 +267,56 @@ func (pubKey PubKey) MarshalAminoJSON() ([]byte, error) {
 func (pubKey *PubKey) UnmarshalAminoJSON(bz []byte) error {
 	return pubKey.UnmarshalAmino(bz)
 }
+
+// Address returns an ethereum style addresses
+func (pubKey *PubKeyOld) Address() crypto.Address {
+	return pubKey.Address()
```

Comment:
Isn't this method calling itself?

Reply:
Oh, forgot to push this 🤦 
[Here](https://github.com/0xPolygon/cosmos-sdk/pull/8/commits/722da0aa266e7f8b721b2556ebd5ec689027ace5) it is, thanks